core/kernel/users/class.Service.php

```
@@ -359,7 +359,7 @@ public function unnatachRole(core_kernel_classes_Resource $user, core_kernel_cla
         } catch (common_Exception $e) {
             $roleUri = $role->getUri();
             $userUri = $user->getUri();
-            $msg = "An error occured while unnataching role '${roleUri}' from user '${userUri}': " . $e->getMessage();
+            $msg = "An error occured while unnataching role '{$roleUri}' from user '{$userUri}': " . $e->getMessage();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$msg'. (undefined)

(UnusedLocalVariable)